server/spec/requests/abouts_spec.rb

```diff
@@ -0,0 +1,7 @@
+require 'rails_helper'
+
+RSpec.describe "Abouts", type: :request do
+  describe "GET /index" do
+    pending "add some examples (or delete) #{__FILE__}"
```

Comment:
Could you add some rspec tests? One suggestion is look missed coverages in coverage report. Report should be created under `server/coverage` once you run `rake spec cucumber`